pages/login.js

```diff
@@ -5,44 +5,62 @@ import Router from "next/router";
 import SkipButton from "../components/skipButton";
 import Input from "../components/formComponents/input";
 import Button from "../components/buttons/button";
+import LoginButton from "../components/buttons/loginButton";
+import { useSession } from "next-auth/react";
 
 export default function Login() {
+  const { data: session } = useSession();
+
   function handleSubmit(event) {
     event.preventDefault();
     Router.push("/dashboard");
   }
 
+  console.log(session);
+
+  if (session) {
+    setTimeout(() => {
+      Router.push("/dashboard");
+    }, 5000);
```

Comment:
Nice feature! Maybe you could display some kind of counter to the user so that they are aware of the fact that they're going to be redirected shortly.